.travis.yml

```diff
@@ -3,8 +3,6 @@ before_script:
   - git config --global user.name "Your Name"
 script: bundle exec rspec
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - rbx
+  - 2.2.3
+  - 2.4.0
```

Comment:
I dropped the Ruby versions because it means less versions for us to support. Overall, I think this decision is still good because there doesn't seem to be any indication of users who are dependent on the dropped versions. If there are, we can restore only those needed.